labrad/test/unit_performance.py

```diff
@@ -132,13 +135,13 @@ def main():
     result = run_tests()
 
     if args.format == 'json':
-        print json.dumps({k: v['ms'] for k, v in result.items()}, indent=2)
+        print(json.dumps({k: v['ms'] for k, v in result.items()}, indent=2))
```

Comment:
This is a side note, but this is a good candidate for switching to constants `ms` instead of `'ms'`